pydocstringformatter/_configuration/toml_parsing.py

```diff
@@ -41,6 +44,28 @@ def parse_toml_option(
         except KeyError:
             raise UnrecognizedOption(f"Don't recognize option {opt}") from exc
 
+    if isinstance(action, BooleanOptionalAction):
+        option_strings = action.option_strings
+
+        if not isinstance(value, bool):
+            error_msg = f"{{'{value}'}} {type(value)} is not a supported argument for"
```

Comment:
```suggestion
            error_msg = f"{{'{value}'}} {type(value)} is not a supported argument for "
```

Reply:
I think the space is needed, we could move it to the next line.

```python
error_msg += f" '{opt}', please use either {{true}} or {{false}}."
#                        ^ here
```

Reply:
I personally like spaces at the end of the line but either way is fine with me! Do what suits you 😄